src/synced_commands.cpp

```diff
@@ -657,7 +657,7 @@ SYNCED_COMMAND_HANDLER_FUNCTION(debug_set_var, child, /*spectator*/)
 	debug_cmd_notification("set_var");
 
 	try {
-		resources::gamedata->set_variable(child["name"],child["value"]);
+		resources::gamedata->set_variable(child["name"], child["value"].t_str());
```

Comment:
maybe set_variable should be changed to take an attribute_value?

Reply:
Good point. I’ll see if that’s feasible.